.github/workflows/ci.yaml

```diff
@@ -69,7 +69,11 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
       - run: npm ci
-      - run: npx bundlesize
+      # Node.js 14 comes with NPM 6 which auto-installs packages by default.
+      - if: ${{ matrix.node == 14 }}
+        run: npx bundlesize
+      - if: ${{ matrix.node != 14 }}
```

Comment:
I think it's enough to compute bundlesizes on a single node version, but equally happy to do them across all of them (not much compute).

Reply:
Good point. It looks like the bundles have identical filenames and sizes on Node.js [14](https://github.com/nextstrain/auspice/actions/runs/4418719566/jobs/7746203399#step:5:51), [16](https://github.com/nextstrain/auspice/actions/runs/4418719566/jobs/7746203686#step:6:6), [18](https://github.com/nextstrain/auspice/actions/runs/4418719566/jobs/7746204105#step:6:6). I'll push a commit to use 18 only, then merge this.